braidpool_spec.md

```diff
@@ -70,18 +82,50 @@ While we could commit to this data in a more space-efficient manner (e.g. via a
 pubkey tweak), the coinbase is also the location of the `extranonce` 8-byte
 field used by some mining equipment.
 
+[kp]
+
+I don't know why you are talking about the consensus mechanism in the above
+paragraph. We are jumping into these details too quickly. Usually you would:
+
+
+1. Identify all components
+2. Provide an over view of all components
+3. Show how the various components together, i.e. the interfaces between them
+4. Finally you dig into each component.
+
+We are doing 4. here straight away. People will be very confused when reading
+this.
+
+[/kp]
+
+
```

Comment:
I think I was trying to get at the difference between committed and uncommitted data, why both exist and the difference between them (which comes down to some being controlled by consensus and some not).